settings.php

```diff
@@ -6,13 +6,82 @@
  */
 
 ?>
-<div class="wrap">
-	<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
-	<form action="options.php" method="post">
+<link rel="stylesheet" href="<?php echo plugin_dir_url( __FILE__ ) . 'static/css/cookiewow.css'; ?>">
+<div class="cookiewow-wrap">
+    <img class="cookiewow-logo" src="<?php echo plugin_dir_url( __FILE__ ) . 'static/images/logo-cookiewow.svg'; ?>" />
+	<form action="options.php" method="post" class="cookiewow-form">
 		<?php
 		settings_fields( 'cookiewow_settings_fields' );
 		do_settings_sections( 'cookiewow-settings' );
 		submit_button();
 		?>
 	</form>
+    <hr />
+    <section class="cookiewow-howto-container">
+        <article class="cookiewow-howto">
+            <h3>
+                Sign in to
+                <a
+                    href="https://cookiewow.com/"
+                    target="_blank"
+                    rel="norefferer"
+                >Cookie Wow</a>
+                (Or sign up
+                    <a
+                        href="https://app.cookiewow.com/users/sign_up"
+                        target="_blank"
+                        rel="norefferer"
+                    >here</a>
+                )
+            </h3>
+            <ol>
+                <li>Add your wordpress domain</li>
+                <li>Go to your domain dashboard and click Customize</li>
+                <li>Copy Banner ID from “Get script” button</li>
+                <li>Paste Banner ID on Wordpress plug-in and click “Save Changes”</li>
```

Comment:
@sugakou44 Typo.

```suggestion
                <li>Paste Banner ID on WordPress plug-in and click “Save Changes”</li>
```

Reply:
fixed in https://github.com/datawowio/cookiewow-wordpress/pull/8/commits/e96287f433845cc3414605847b15de70478351b5